samples/HotAddSample/Startup.cs

```diff
@@ -12,11 +14,21 @@ namespace HotAddSample
     // will be reset before the end of the request.
     public class Startup
     {
+        public void ConfigureServices(IServiceCollection services)
```

Comment:
Curious why the new way of resolving the url addresses? What was wrong with the original approach?